dice-game-ui-angular/src/app/saved-games/saved-games.component.html

```diff
@@ -0,0 +1,24 @@
+<h1 align="center" class="text-muted">Saved games</h1>
+
+<div *ngIf="games.length>0; else noSavedGames">
+    <mat-card class="m-1" *ngFor="let ge of games">
+        <mat-card-header>
+            <mat-card-title>
+                <button mat-flat-button [routerLink]="['/game', ge.guid]">
+                    Single game {{ge.gameEngine.fieldSize.width}} x {{ge.gameEngine.fieldSize.height}} from {{ge.creationDate|date:"dd.MM.yyyy"}}
+                </button>
+            </mat-card-title>
+        </mat-card-header>
+        <mat-card-actions align="end">
+            <button mat-mini-fab color="primary" align="start" [routerLink]="['/game', ge.guid]" aria-label="Continue" class="m-1">
+                <mat-icon>login</mat-icon>
+            </button>
+            <button mat-mini-fab color="accent" (click)="removeSavedGame(ge.guid)" aria-label="Remove" class="m-1">
```

Comment:
You'd better pass whole game object to the method.

Reply:
Why? The actual method doesn't need it.

Reply:
Because method implementation could change and in my case it won't lead to changes in a template, only in component code. But it's up to you.

Reply:
Ok, then let's leave it "as is" because now I don't see any cases when the method removing saved game engine needs the whole object.